lib/geordi/gitpt.rb

```diff
@@ -54,6 +61,8 @@ def choose_story
     stories = applicable_stories
     reset_loading_message = "\r#{' ' * (loading_message.length + stories.length)}\r"
 
+    Geordi::Interaction.fail('No stories found') if stories.empty?
```

Comment:
Please fail with a sentence, and change wording:
```suggestion
    Geordi::Interaction.fail('No stories to offer.') if stories.empty?
```